### DISCUSSION TITLE
新艾利都的大家好啊♪

### DISCUSSION THREAD

**Discussion by Elysia-lover** [share121/inter-knot#895] [其他]
嗨，你们好啊♪我是新来的Elysia，可以直接叫我爱莉♪
可爱的Elysia为了找到走失的妹妹小蚀，听了傲娇的小蛇奶奶来到这个美丽的新艾利都♪嗯，这个城市名字很有品味，我喜欢♪到时候给伊甸捎点好看的照片过去♪
总之，大家多多关照吧♪
对了，我喜欢漂亮的女孩子哦♪
![1714213376287](https://github.com/user-attachments/assets/aec489e1-f836-4d83-b7f3-a25661fecb7a)

**Comment by AurEian** [comment]
为什么爱莉会出现在这里呢?因为这里是新艾利(爱莉)都

**Comment by SixRadium** [comment]
大风机关打进绳网了！
